src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,70 @@
+package duke;
```

Comment:
I like how you have organised the exceptions, commands and tasks into their own sub packages of duke.